src/components/Drawer.jsx

```diff
@@ -198,23 +188,16 @@ Drawer.propTypes = {
   onClose: PropTypes.func,
   onLogOut: PropTypes.func,
   isInvertedTheme: PropTypes.bool,
-  storageData: PropTypes.object,
-  fetchSettingsData: PropTypes.func,
-  logOut: PropTypes.func
+  fetchSettingsData: PropTypes.func
 }
 
 export { Drawer }
 
-const mapStateToProps = state => ({
-  storageData: getStorageData(state)
-})
-
 const mapDispatchToProps = dispatch => ({
-  fetchSettingsData: () => dispatch(fetchSettingsData()),
-  logOut: () => dispatch(logOut())
+  fetchSettingsData: () => dispatch(fetchSettingsData())
```

Comment:
Do you still need this dispatch as you are using `useInstanceInfo` directly inside the component below ?

Reply:
je vais regarder et corriger si nécessaire